utils/build-script-impl

```diff
@@ -2378,6 +2402,21 @@ for host in "${ALL_HOSTS[@]}"; do
                   -DBUILD_SHARED_LIBS=$([[ ${product} == foundation_static ]] && echo "NO" || echo "YES")
                 )
 
+                if [[ $(is_cross_tools_host ${host}) ]] ; then
+                    cmake_options+=("${SWIFT_TARGET_CMAKE_OPTIONS[@]}")
+
+                    # CROSS_COMPILE_DEPS_PATH is searched for the ICU, libXML2
+                    # and libcurl dependencies.
+                    cmake_options+=(
+                        -DCMAKE_FIND_ROOT_PATH:PATH="${CROSS_COMPILE_DEPS_PATH}"
+                        -DCMAKE_FIND_ROOT_PATH_MODE_PROGRAM=NEVER
```

Comment:
Similar here. I just think since these are not used that often. I would add a small comment explaining what these specific CMAKE options are needed with maybe a link to the cmake docs.

Reply:
Sure.